requirements.dev.txt

```
@@ -17,5 +17,6 @@ pyyaml==5.1
 git+git://github.com/hannes-ucsc/requirements-parser@v0.2.0+1#egg=requirements-parser
 responses==0.10.2
 watchdog==0.9.0
+git+git://github.com/DataBiosphere/azul-tinyquery@tags/1.2+1#egg=Tinyquery
```

```suggestion
git+git://github.com/DataBiosphere/azul-tinyquery@tags/1.2+1#egg=tinyquery
```

Any idea why you need `tags/` but the other "git+git:" dependencies don't?

---

It works without `tags/` now that the version is correct.